core/state_processor.go

```diff
@@ -66,6 +66,7 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 		blockNumber = block.Number()
 		allLogs     []*types.Log
 		gp          = new(GasPool).AddGas(block.GasLimit())
+		beaconRoot  *common.Hash
```

Comment:
Is this just a placeholder until we later activate this by checking `block.BeaconRoot`?

Reply:
Right, yeah we should remove this and use block.BeaconRoot instead. Looks very odd like that

Reply:
Ah, yeah that change is probably in the follow-up PR

Reply:
Hm, https://github.com/ethereum/go-ethereum/pull/27872 is the follow-up, but it still doesn't fill this placeholder :/

Reply:
@lightclient thanks for adding your updates, I have also pushed some changes now -- primarily, hooking it into `evm t8n` so I could actually test it and see it working in practice. 
PTAL